src/lib.rs

```diff
@@ -114,13 +113,63 @@ pub fn select_coin_lowestlarger(
 }
 
 /// Perform Coinselection via First-In-First-Out algorithm.
+
 /// Return NoSolutionFound, if no solution exists.
+
 pub fn select_coin_fifo(
     inputs: Vec<OutputGroup>,
     options: CoinSelectionOpt,
-    excess_strategy: ExcessStrategy,
 ) -> Result<SelectionOutput, SelectionError> {
-    unimplemented!()
+    /* Using the value of the input as an identifier for the selected inputs, as the index of the vec<outputgroup> can't be used because the vector itself is sorted first. Ideally, txid of the input can serve as an unique identifier */
```

Comment:
This is wrong. The indices of the Vec has to be the identifier. You can create a Vec<usize, OutGroup> first and then sort that Vec, this will track which OutGroup belongs to which index.
Mulltiple UTXOs can have same OutGroup value.

Reply:
Right. Removed the comment.